samples/validation/README.md

```diff
@@ -0,0 +1,6 @@
+# Validation
+
+Sample to demonstrate the validation helpers of the user_settings lib.
```

Comment:
Please briefly describe how the sample does this. I see that the shell is enabled, so this file should atleast say that is expected that the user should check the source code to see what is the expected range (or expected exact size) and then try changing that setting over the shell.

Reply:
I will clean up in a new PR.